package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0",
+  "version": "2.1.0",
```

Comment:
Can you make this 1.1.0

Reply:
Or 2.0.0

Reply:
I started by making it 1.1.0 but then I noticed the npm package was at
2.0.0 currently. I can set it to whatever you like, though. Just confirm what that should be.